eZ/Bundle/EzPublishCoreBundle/ApiLoader/RepositoryFactory.php

```diff
@@ -47,18 +48,23 @@ class RepositoryFactory implements ContainerAwareInterface
     /** @var \eZ\Publish\API\Repository\LanguageResolver */
     private $languageResolver;
 
+    /** @var \eZ\Publish\API\Repository\PermissionResolver */
+    private $permissionResolver;
+
     public function __construct(
         ConfigResolverInterface $configResolver,
         $repositoryClass,
         array $policyMap,
         LanguageResolver $languageResolver,
+        PermissionResolver $permissionResolver,
```

Comment:
@Nattfarinn this whould be injected into `buildRepository` factory only (reflected in #46)